core/dbt/tests/util.py

```diff
@@ -430,3 +435,46 @@ def check_table_does_not_exist(adapter, name):
 def check_table_does_exist(adapter, name):
     columns = get_relation_columns(adapter, name)
     assert len(columns) > 0
+
+
+# Utility classes for enabling comparison of dictionaries
+
+
+class AnyFloat:
+    """Any float. Use this in assert calls"""
+
+    def __eq__(self, other):
+        return isinstance(other, float)
+
+
+class AnyInteger:
+    """Any Integer. Use this in assert calls"""
+
+    def __eq__(self, other):
+        return isinstance(other, int)
+
+
+class AnyString:
+    """Any string. Use this in assert calls"""
+
+    def __eq__(self, other):
+        return isinstance(other, str)
```

Comment:
The three classes here looks very similar, why we are doing it as 3 individual instead of having one that you can maybe pass in a specific type? Also, think we can achieve `AnyString` by not passing in contains for `AnyStringWith`?

Reply:
They're separate classes because they were copied from existing classes in the integration test suite. You do have a point that a single class that takes a type would work here.